### ISSUE TITLE
Flutter Pdfviewer Fatal error: Unexpectedly found nil while unwrapping an Optional value Swift:128

### ISSUE THREAD

**Issue by Taimoorsikander** [syncfusion/flutter-examples#768]
Hi, I am using Flutter Pdfviewer, and it's working perfectly in any case but in iOS while the pdf is loading, I press the back button to close the screen or change it than the app crashes. This is the error that I'm getting
syncfusion_flutter_pdfviewer/SwiftSyncfusionFlutterPdfViewerPlugin.swift:128: Fatal error: Unexpectedly found nil while unwrapping an Optional value

**Comment by deepika134** [comment]
We tried to reproduce the reported issue with provided details, but the PDF loads properly without any crashes in our side. For your reference, we have prepared a simple sample (along with the output scree record of the same) and it can be downloaded from the attachment.

If you still reproduce the reported issue in your use case, then kindly share the modified issue reproducing in the sample and share the issue replicating video, which will be helpful for us to provide a prompt solution at the earliest.
[github.zip](https://github.com/syncfusion/flutter-examples/files/11403160/github.zip)
